doc/api/v0.yaml

```diff
@@ -171,6 +172,11 @@ paths:
           schema:
             type: integer
           required: true
+        - in: path
+          name: voter_group_id
```

Comment:
isn't this missing in the path? and btw I don't why it should be required

Reply:
Yes, it is missing. Speaking of why should it be required: well, you've got a point, we may leave the possibility to get all proposals regardless of groups.